server/controllers/contest.js

```diff
@@ -85,4 +86,31 @@ exports.getContests = asyncHandler(async (req, res, next) => {
     } catch (error) {
     return res.status(500).send({ error });
     }
+})
+
+
+exports.createSubmission = asyncHandler(async (req, res, next) => {
+    const userID = req.user.id;
+    const contestID = req.params.id;
+    const { imageFiles } = req.body;
+    const active = true;
```

Comment:
The field `active` is true by default, we don't need to add this while creating a submission :)